scripts/install2

```diff
@@ -88,6 +88,8 @@ fi
 
 echo -e "\e[1;32mStarting installation on ${BOARD_FAMILY} ${BOARD_NAME} (${BOARD_REVISION})...\e[0m"
 
+# Normalize OS environment
+
```

Comment:
The extra space here separates the SECTION comment from the next line comment

IE: This section is to normalize the OS  environment